source/fundamentals/auth.txt

```diff
@@ -48,38 +48,31 @@ Mechanisms
 ----------
```

Comment:
I: can't comment directly, but you can remove lines 29 and 32 (ref links to the "Default" and "MONGODB-CR" mechanisms in the overview list)